test/hotspot/jtreg/runtime/Thread/AsyncExceptionOnMonitorEnter.java

```diff
@@ -76,21 +76,23 @@ public void testWithJavaMonitor() {
 
 
     public void testWithJVMTIRawMonitor() {
-        started = true;
         try {
+            started = true;
             int retCode = enterRawMonitor();
             if (retCode != 0) {
                 throw new RuntimeException("error in JVMTI RawMonitorEnter: retCode=" + retCode);
             }
             gotMonitor = true;
-            Thread.sleep(500);
+            Thread.sleep(1000);
             retCode = exitRawMonitor();
             if (retCode != 0) {
                 throw new RuntimeException("error in JVMTI RawMonitorExit: retCode=" + retCode);
             }
         } catch (ThreadDeath td) {
         } catch (InterruptedException e) {
             throw new Error("Unexpected: " + e);
+        } finally {
+            exitRawMonitor();
```

Comment:
This isn't quite right. If we have the exit in the finally block then it should be deleted from the main block. But we also need the enter to be outside the scope of the try-catch-finally incase we abort before entering the raw monitor. Not worth wasting time on this part - just restore it to how it was.